Automattic/simplenote-macos
# Updated the intents extension version number

### Fix
The version number on the new intents extension wasn't set correctly.  This PR updates that so that the version number is the same as the main target.

### Test
1. Go to the project navigator in Xcode, select the IntentsExtension -> Build Settings and filter for `Version`
2. Confirm that the marketing version is 2.20 and the project version is 11258

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
***(Required)*** Add a concise statement to `RELEASE-NOTES.txt` if the changes should be included in release notes. Include details about updating the notes in this section. For example:
> `RELEASE-NOTES.txt` was updated in d3adb3ef with:
> 
> > Added markdown support

If the changes should not be included in release notes, add a statement to this section. For example:

> These changes do not require release notes.
